qualtran/bloqs/mcmt/multi_control_multi_target_pauli.py

```diff
@@ -111,43 +101,61 @@ class MultiControlPauli(GateWithRegisters):
     target_gate: cirq.Pauli = cirq.X
 
     @cached_property
-    def signature(self) -> Signature:
-        return Signature.build(controls=len(self.cvs), target=1)
+    def signature(self) -> 'Signature':
+        ctrl = Register('controls', QBit(), shape=(len(self.cvs),))
+        target = Register('target', QBit())
+        return Signature([ctrl, target] if len(self.cvs) > 0 else [target])
```

Comment:
why are we supporting no controls?

Reply:
There was an explicit feature request from Lincoln lab folks. See https://github.com/quantumlib/Qualtran/issues/677